qute.ls/com.redhat.qute.ls/src/main/resources/com/redhat/qute/project/datamodel/qute-resolvers.json

```diff
@@ -25,6 +25,23 @@
 				"{#for r in recordsList.take(3)}"
 			],
 			"url": "https://quarkus.io/guides/qute-reference#collections"
+		},
+		{
+			"signature": "or(self: T, arg : java.lang.Object) : boolean",
+			"description": "Returns the default value if the base object is null, empty or not found and the base Object otherwise",
+			"sample": [
+				"{item.price.or(5)}",

```

Comment:
For sample please use the same sample than reference guide:

"sample": [
"{person.name ?: 'John'}", 
"{person.name or 'John'}", 
"{person.name.or('John')}"
]